pkg/controller/login/reset_password.go

```diff
@@ -59,6 +60,11 @@ func (c *Controller) HandleSubmitResetPassword() http.Handler {
 			return
 		}
 
+		membership := controller.MembershipFromContext(ctx)
```

Comment:
I don't think we require that a current realm is select for the user to reset their own password.